cmd/prepare-storage-msg_test.go

```diff
@@ -18,9 +18,38 @@ package cmd
 
 import (
 	"net/url"
+	"reflect"
 	"testing"
 )
 
+func TestHealEndpoint(t *testing.T) {
```

Comment:
since this function tests `getHealEndpoint` only it should be named TestGetHealEndpoint since that is the convention